index.js

```diff
@@ -67,6 +68,20 @@ function throttle(options) {
         throw new Error('(ip ^ username ^ xff)');
     }
 
+    for (key in options.overrides) {
+        var override = options.overrides[key];
+        try {
+            var block = new Netmask(key);
+
+            // Non-/32 blocks only
+            if (block.first !== block.last) {
+                override.block = block;
+            }
+        }
+        catch(err) {
```

Comment:
A key can be a username from `req.username`, which would fail, but should probably be ignored.